### PR TITLE
fix: updated remaining links that pointed to github.io/node-newrelic.…

### PR DESCRIPTION
**Pull request by bizob2828**
## Give us some context

* What problems does this PR solve?
In #11114 my global find and replace constrained the links that were prefixed with `https://`.  I checked our docs today and realized I missed a bunch.  This PR resolves the remaining ones.

### CODE REVIEW

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ Other resources:
 To use the Node.js agent API, make sure you have the [latest Node.js agent release](/docs/release-notes/agent-release-notes/nodejs-release-notes). In addition, see:
 
 * [Node.js agent API requirements](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api)
-* [Getting started procedures on GitHub](http://newrelic.github.io/node-newrelic/docs/index.html#getting-started)
+* [Getting started procedures on GitHub](http://newrelic.github.io/node-newrelic/index.html#getting-started)
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:30]: @bizob2828 For some reason this anchor tag link doesn't work. As far as I can tell from the HTML code on this GitHub.io page, the headers don't actually have IDs, which means that can't be linked to directly.

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 2]
```diff
@@ -412,6 +412,6 @@ You can also control the browser agent via APM agent API calls. For more informa
 
 ## Extend custom instrumentation [#custom-instrumentation]
 
-The `newrelic.instrument()` provides additional flexibility for custom instrumentation. For more information, including tutorials and examples, see the [shims documentation on GitHub](http://newrelic.github.io/node-newrelic/docs/Shim.html).
+The `newrelic.instrument()` provides additional flexibility for custom instrumentation. For more information, including tutorials and examples, see the [shims documentation on GitHub](http://newrelic.github.io/node-newrelic/Shim.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:415]:
```suggestion
The `newrelic.instrument()` provides additional flexibility for custom instrumentation. For more information, including tutorials and examples, see the [shim documentation](http://newrelic.github.io/node-newrelic/Shim.html)  on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 3]
```diff
@@ -308,7 +308,7 @@ Once the [request naming API](/docs/agents/nodejs-agent/supported-features/nodej
         * [Custom instrumentation API](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#custom-instrumentation-api)
         * [Message queues](/docs/agents/nodejs-agent/supported-features/message-queues)
 
-          Also see the tutorials on GitHub for [datastore shims](http://newrelic.github.io/node-newrelic/docs/DatastoreShim.html) and [message shims](http://newrelic.github.io/node-newrelic/docs/MessageShim.html).
+          Also see the tutorials on GitHub for [datastore shims](http://newrelic.github.io/node-newrelic/DatastoreShim.html) and [message shims](http://newrelic.github.io/node-newrelic/MessageShim.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:311]:
```suggestion
          Also see the [datastore](http://newrelic.github.io/node-newrelic/DatastoreShim.html) and [message shim](http://newrelic.github.io/node-newrelic/MessageShim.html) tutorials on GitHub for .
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx (hunk 1 above)
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:30]:
```suggestion
* [Getting started procedures](http://newrelic.github.io/node-newrelic/index.html#getting-started) on GitHub
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 4]
```diff
@@ -68,7 +68,7 @@ Use these methods when New Relic is not instrumenting a particular part of your
         Use either of these options:
 
         * Return a promise from the callback handed to [`newrelic.startWebTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#startWebTransaction).
-        * Call `end` on a [handle](http://newrelic.github.io/node-newrelic/docs/API.html#getTransaction) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
+        * Call `end` on a [handle](http://newrelic.github.io/node-newrelic/API.html#getTransaction) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:71]:
```suggestion
        * Call `end` on a [handle](http://newrelic.github.io/node-newrelic/API.html#getTransaction) (GitHub) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 5]
```diff
@@ -81,7 +81,7 @@ Use these methods when New Relic is not instrumenting a particular part of your
         Ignore the transaction using any of these options:
 
         * See [Rules for ignoring requests](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#ignoring)`.`
-        * Call `ignore()` on a [handle](http://newrelic.github.io/node-newrelic/docs/TransactionHandle.html) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
+        * Call `ignore()` on a [handle](http://newrelic.github.io/node-newrelic/TransactionHandle.html) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:84]:
```suggestion
        * Call `ignore()` on a [handle](http://newrelic.github.io/node-newrelic/TransactionHandle.html) (GitHub) returned from [`newrelic.getTransaction`](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#getTransaction).
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 6]
```diff
@@ -119,7 +119,7 @@ Use this method when you want to instrument a method within an existing transact
   </tbody>
 </table>
 
-For more information about timing, see the [instrumentation tutorial on GitHub](http://newrelic.github.io/node-newrelic/docs/tutorial-Instrumentation-Basics.html).
+For more information about timing, see the [instrumentation tutorial on GitHub](http://newrelic.github.io/node-newrelic/tutorial-Instrumentation-Basics.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:122]:
```suggestion
For more information about timing, see the [instrumentation tutorial](http://newrelic.github.io/node-newrelic/tutorial-Instrumentation-Basics.html)  on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx [hunk 7]
```diff
@@ -27,15 +27,15 @@ The custom instrumentation methods in this document are available as of [Node.js
 
 ## Instrument unsupported web frameworks [#web-framework]
 
-Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional web frameworks. For more information, including a tutorial, see the documentation for [Node.js web framework instrumentation on GitHub](http://newrelic.github.io/node-newrelic/docs/tutorial-Webframework-Simple.html).
+Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional web frameworks. For more information, including a tutorial, see the documentation for [Node.js web framework instrumentation on GitHub](http://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx:30]:
```suggestion
Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional web frameworks. For more information, including a tutorial, see the documentation for the [Node.js web framework instrumentation](http://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html) on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx [hunk 8]
```diff
@@ -27,15 +27,15 @@ The custom instrumentation methods in this document are available as of [Node.js
...
 
 ## Instrument unsupported message service clients [#message-client]
 
-Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional message service libraries. For more information, including a tutorial, see the documentation for [Node.js message service client instrumentation on GitHub](http://newrelic.github.io/node-newrelic/docs/tutorial-Messaging-Simple.html).
+Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional message service libraries. For more information, including a tutorial, see the documentation for [Node.js message service client instrumentation on GitHub](http://newrelic.github.io/node-newrelic/tutorial-Messaging-Simple.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx:34]:
```suggestion
Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional message service libraries. For more information, including a tutorial, see the documentation for the [Node.js message service client instrumentation](http://newrelic.github.io/node-newrelic/tutorial-Messaging-Simple.html) on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx [hunk 9]
```diff
@@ -27,15 +27,15 @@ The custom instrumentation methods in this document are available as of [Node.js
...
 
 ## Instrument unsupported datastores [#datastore]
 
-Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional datastore libraries. For more information, including a tutorial, see the documentation for [Node.js datastore instrumentation on GitHub](http://newrelic.github.io/node-newrelic/docs/tutorial-Datastore-Simple.html).
+Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional datastore libraries. For more information, including a tutorial, see the documentation for [Node.js datastore instrumentation on GitHub](http://newrelic.github.io/node-newrelic/tutorial-Datastore-Simple.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx:38]:
```suggestion
Beginning with Node.js agent version 2.0.0, New Relic provides an API to expand instrumentation for additional datastore libraries. For more information, including a tutorial, see the documentation for the [Node.js datastore instrumentation](http://newrelic.github.io/node-newrelic/tutorial-Datastore-Simple.html) on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx [hunk 10]
```diff
@@ -132,7 +132,7 @@ In order to create custom [web transactions](/docs/apm/transactions/intro-transa
     });
     ```
 
-    This method only gives basic timing data for the transaction created. To create more intricate timing data and transaction naming for a particular framework, see the [Node.js API docs](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#instrumentWebframework) and the [related tutorial on GitHub](http://newrelic.github.io/node-newrelic/docs/tutorial-Webframework-Simple.html).
+    This method only gives basic timing data for the transaction created. To create more intricate timing data and transaction naming for a particular framework, see the [Node.js API docs](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#instrumentWebframework) and the [related tutorial on GitHub](http://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html).
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation.mdx:135]:
```suggestion
    This method only gives basic timing data for the transaction created. To create more intricate timing data and transaction naming for a particular framework, see the [Node.js API docs](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api#instrumentWebframework) and the [related WebFramework tutorial](http://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html) on GitHub.
```

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx (hunk 1 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx]: we must've changed structure around. I can remove the anchor

Comment (paperclypse) [reply src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx]: I'm happy to do it.

File: src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx [hunk 11]
```diff
@@ -27,7 +27,7 @@ Other resources:
 To use the Node.js agent API, make sure you have the [latest Node.js agent release](/docs/release-notes/agent-release-notes/nodejs-release-notes). In addition, see:
 
 * [Node.js agent API requirements](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api)
-* [Getting started procedures on GitHub](http://newrelic.github.io/node-newrelic/docs/index.html#getting-started)
+* [Getting started procedures](http://newrelic.github.io/node-newrelic/index.html#getting-started) on GitHub
```
Comment (paperclypse) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx:30]:
```suggestion
* [Getting started procedures](http://newrelic.github.io/node-newrelic/index.html) on GitHub
```

**Comment by paperclypse** [comment]
@bizob2828 As an FYI and to save you some time next time, you don't need to update Japanese or Korean or any other translated content. That content is updated as part of our automated translation process. 

Thanks for the PR! I'll get it merged in once the checks finish.

**Comment by bizob2828** [comment]
> @bizob2828 As an FYI and to save you some time next time, you don't need to update Japanese or Korean or any other translated content. That content is updated as part of our automated translation process. 
> 
> 
> 
> Thanks for the PR! I'll get it merged in once the checks finish.

Good to know. I just did a global find and replace so no saved time for me 😞

**Pull request merged**